src/constants/index.ts

```diff
@@ -191,23 +195,172 @@ export const SUPPORTED_WALLETS: { [key: string]: WalletInfo } = {
+      JSBI.BigInt(1000),
+      GlobalConst.utils.BIPS_BASE,
+    ), // 10%
+    BLOCKED_PRICE_IMPACT_NON_EXPERT: new Percent( // for non expert mode disable swaps above this
+      JSBI.BigInt(1500),
+      GlobalConst.utils.BIPS_BASE,
+    ), // 15%
+  },
+  tokens: {
+    MATIC: WETH[ChainId.MATIC],
+    COMMON: {
+      EMPTY: new Token(
+        ChainId.MATIC,
+        '0x0000000000000000000000000000000000000000',
+        0,
+        'EMPTY',
+        'EMPTY',
+      ),
+      USDC: new Token(
+        ChainId.MATIC,
+        '0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174',
+        6,
+        'USDC',
+        'USDC',
+      ),
+      USDT: new Token(
+        ChainId.MATIC,
+        '0xc2132D05D31c914a87C6611C10748AEb04B58e8F',
+        6,
+        'USDT',
+        'Tether USD',
+      ),
+      OLD_QUICK: new Token(
+        ChainId.MATIC,
+        GlobalConst.addresses.QUICK_ADDRESS,
+        18,
+        'QUICK(OLD)',
+        'Quickswap(OLD)',
+      ),
+      NEW_QUICK: new Token(
+        ChainId.MATIC,
+        GlobalConst.addresses.NEW_QUICK_ADDRESS,
+        18,
+        'QUICK(NEW)',
+        'QuickSwap(NEW)',
+      ),
+      OLD_DQUICK: new Token(
+        ChainId.MATIC,
+        '0xf28164A485B0B2C90639E47b0f377b4a438a16B1',
+        18,
+        'dQUICK',
+        'Dragon QUICK',
+      ),
+      NEW_DQUICK: new Token(
+        ChainId.MATIC,
+        '0x958d208Cdf087843e9AD98d23823d32E17d723A1',
+        18,
+        'dQUICK',
+        'Dragon QUICK',
+      ),
+      WBTC: new Token(
+        ChainId.MATIC,
+        '0x1BFD67037B42Cf73acF2047067bd4F2C47D9BfD6',
+        18,
+        'wBTC',
+        'Wrapped Bitcoin',
+      ),
+      DAI: new Token(
+        ChainId.MATIC,
+        '0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063',
+        18,
+        'DAI',
+        'Dai Stablecoin',
+      ),
+      ETHER: new Token(
+        ChainId.MATIC,
+        '0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619',
+        18,
+        'ETH',
+        'Ether',
+      ),
+      CXETH: new Token(
+        ChainId.MATIC,
+        '0xfe4546feFe124F30788c4Cc1BB9AA6907A7987F9',
+        18,
+        'cxETH',
+        'CelsiusX Wrapped ETH',
+      ),
+      MI: new Token(
+        ChainId.MATIC,
+        '0xa3Fa99A148fA48D14Ed51d610c367C61876997F1',
+        18,
+        'MAI',
+        'miMATIC',
+      ),
+      SAND: new Token(
+        ChainId.MATIC,
+        '0xBbba073C31bF03b8ACf7c28EF0738DeCF3695683',
+        18,
+        'SAND',
+        'SAND',
+      ),
+      MAUSDC: new Token(
+        ChainId.MATIC,
+        '0x9719d867A500Ef117cC201206B8ab51e794d3F82',
+        6,
+        'maUSDC',
+        'Matic Aave interest bearing USDC',
+      ),
+      FRAX: new Token(
+        ChainId.MATIC,
+        '0x45c32fA6DF82ead1e2EF74d17b76547EDdFaFF89',
+        18,
+        'FRAX',
+        'FRAX',
+      ),
+    },
+    UNI: {
```

Comment:
I think this is not necessary and we can remove this value. We are using this value in the hooks for farms/syrups in `stake/hooks.ts` and `useUniContract` in `hooks/useContract.ts` which is not used and can be removed.